src/main/kotlin/io/snyk/plugin/services/SnykTaskQueueService.kt

```diff
@@ -74,16 +81,69 @@ class SnykTaskQueueService(val project: Project) {
 
                 if (settings.ossScanEnable) {
                     if (!getOssService(project).isCliInstalled()) downloadLatestRelease()
```

Comment:
shell we move isCliInstalled to Utils and do such check product agnostic?
also probbaly that need/could be inside waitForCliDownload()

Reply:
It is just in the base class right now - but yeah, I was wondering why it was service-dependent, but didn't want to change too much ;).